### PR TITLE
Refactor breath bubbles to use lit approach

### PR DESCRIPTION
**Pull request by u9g**
Refactors breath bubbles so they take advantage of the lit rendering engine rather than doing the updates through code

### CODE REVIEW

**Comment by u9g** [comment]
https://streamable.com/ubm7cw

**Comment by rom1504** [comment]
Looks good
CI failing

**Review by AwesomestCode — CHANGES_REQUESTED** [review]
Code looks good for the most part but it seems to be a bit buggy. Testing on a lake, when oxygen is exhausted (where you're already losing HP) it keeps flashing between full and to empty. Also, when you go up to get air sometimes you'll still have oxygen flashing, not sure why.

**Comment by rom1504** [comment]
old, re-open if you want to finish

**Comment by rom1504** [comment]
still old, but can stay open if you want to fix it

File: lib/menus/components/breath_bar.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ class BreathBar extends LitElement {
     `
   }
 
-  static properties = {
+  static get properties = {
```
Comment (rom1504) [inline lib/menus/components/breath_bar.js]: you're missing the parenthesis after properties

**Comment by rom1504** [comment]
too old, please re-open if you want to finish

**Pull request closed without merging**